src/logplex_logging.hrl

```diff
@@ -6,14 +6,30 @@
 -define(logging_macros, true).
 -compile([{parse_transform, lager_transform}]).
 
+%% Uncomment this to use batchio logging
+-define(LOG_TO_SYSLOG, true).
+
+-define(LOGGING_ARGS(LEVEL, FORMAT, ARGS),
+        "pid=~p m=~p ln=~p class=~s " ++ FORMAT ++ "~n", [self(), ?MODULE, ?LINE, LEVEL | ARGS]).
```

Comment:
since the syslog_lib module seems to accept an iolist an optimization might be to avoid using the `++` operator twice.

```erlang
iolist_to_binary(["pid=~p m=~p ln=~p class=~s ", FORMAT, "~n"], [self(), ?MODULE, ?LINE, LEVEL | ARGS]
```